content/guest-post/secure-php-api-using-jwt/index.md

```diff
@@ -0,0 +1,254 @@
+```
+The above string is an example of a JWT authentication string. At first glance, it may appear to be a randomly produced string. But don't underestimate; this string is made up of three separate components that are essential in a JWT.
+
+### JWT Header
+The header of a JWT is the initial section of the string before the first dot. This header is produced by acquiring plain text and performing cryptographic operations on it. Moreover, the header uses a very efficient Base64 encoding procedure.
+
+You can easily obtain the JWT's headers using symmetric or asymmetric encryption techniques.
+
+### JWT Payload
+The string's central component is the JWT's payload part. This string includes all of the important information about a received request and the user or client computer who created the request. There are predefined key-value pair fields in the payload that can be used to offer extra information about the received request. Here is an explanation of common payload fields.
+
+- **Sub** - The sub field contains the subject of a JWT payload. It contains unique information about the user and client device that has created this authentication request.
+- **Iss** - This field contains data about the server that has issued the token. Iss is short for Issuer, which refers to the server.
+- **Exp** - Unlike other authentication techniques, JWT has an expiration time. This field's name is a short form for the expiration date. It contains data about the time when the token was issued and the expiration date and time of the issued token.
+
+### JWT Signature
+A cryptographic operation is performed on the JWT data to obtain this signature. It takes in the payload, secret key, and header value of a JWT. The signature is then generated by applying a function to these obtained values.
+
+The server and user can verify this signature to know about the data's security and integrity. If this signature matches at both ends, then the data is considered secure, and all other transactions can take place.
+
+## Using JWTs to Secure PHP API
+As you've understood everything about JWT, let's secure your PHP API using JWT. Follow the code along, and you'll have a tamper-proof PHP API.
+
+This article creates a simple login page and authenticates it using JWT. This will help you get started with JWT and PHP.
+
+To follow along, you'll need to have PHP and composer installed on your computer.
+
+If you haven't already installed composer on your computer, you may do so here. Once the composer has been installed, run the tool from your project folder. Composer will assist you in installing Firebase PHP-JWT, a third-party library for working with JWTs and Apache.
```

Comment:
Kindly add a link to `here` which leads to  installation page of composer.

Reply:
https://getcomposer.org/doc/00-intro.md